src/main/java/Duke.java

```diff
@@ -1,10 +1,585 @@
+                "\nCOMMANDS: \n" +
+                "* list -- prints a numbered list of the tasks created, in input order.\n" +
+                "* mark [] -- marks the task at index [] as completed. \n" +
+                "* unmark [] -- marks the task at index [] as uncompleted. \n" +
+                "* delete [] -- removes the task at index [] from the list.\n" +
+                "* bye -- exits the program.");
+    }
+
+    /**
+     * Prints Signal's response enclosed in the dividers.
+     *
+     * @param response The message that is printed.
+     */
+    public static void signalSays(String response) {
+        System.out.println(DIV);
+        System.out.println(response);
+        System.out.println(DIV);
+    }
+
+
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! My name is\n" + LOGO);
+        System.out.println("How can I help?");
+        System.out.println("Enter 'help' to see the list of commands available :D");
+        System.out.println(DIV);
+
+
+        while(true) {
+            String userInput = scanner.nextLine();
+            String[] inputArray = userInput.split(" ");
+
+            if (userInput.equals("bye")) {
+                // Exit program
+                break;
+            } else if (userInput.equals("")) {
+                // input is blank
+                signalSays("Brevity is the soul of wit, but you have to tell me something still!");
+            } else if (inputArray.length == 2 && inputArray[0].equals("mark")) {
+                // Mark item at index as done
+                int itemIndex = Integer.parseInt(inputArray[1]) - 1;
+                signalSays(commandMark(itemIndex));
+            } else if (inputArray.length == 2 && inputArray[0].equals("unmark")) {
+                // Mark item at index as done
+                int itemIndex = Integer.parseInt(inputArray[1]) - 1;
+                signalSays(commandUnmark(itemIndex));
+            } else if (inputArray.length == 2 && (isPermutationMatch(inputArray[0], "mark") || isPermutationMatch(inputArray[0], "unmark"))) {
+                // check typo of "mark" or "unmark"
+                if (checkCommandTypo(inputArray[0], "mark")) { // command mark typo
+                    int itemIndex = Integer.parseInt(inputArray[1]) - 1;
+                    signalSays(commandMark(itemIndex));
+                } else if (checkCommandTypo(inputArray[0], "unmark")) { // command unmark typo
+                    int itemIndex = Integer.parseInt(inputArray[1]) - 1;
+                    signalSays(commandUnmark(itemIndex));
```

Comment:
I don't fully understand the purpose of `checkCommandTypo`. Isn't it always going to trigger when you type "unmark" since it checks for "mark" first?